s3/pkg/meta/db/drivers/tidb/transation.go

```diff
@@ -0,0 +1,18 @@
+package tidbclient
+
+import "database/sql"
+
+func (t *TidbClient) NewTrans()(tx interface{}, err error) {
```

Comment:
please define return values in case of error...like transaction cannot start etc.